openmdao/core/driver.py

```diff
@@ -70,6 +70,13 @@ def _setup(self, root):
         for item_name, item, newitem in item_tups:
             for name, meta in iteritems(item):
                 rootmeta = root.unknowns.metadata(name)
+                
+                if name in self._desvars:
+                    root.unknowns.metadata(name)['is_desvar'] = True
```

Comment:
Ooops. Fixed.

On Nov 4, 2015, at 12:56 PM, Bret Naylor <notifications@github.com<mailto:notifications@github.com>>
 wrote:


In openmdao/core/driver.py<https://github.com/OpenMDAO/OpenMDAO/pull/378#discussion_r43914562>:

> @@ -70,6 +70,13 @@ def _setup(self, root):
>          for item_name, item, newitem in item_tups:
>              for name, meta in iteritems(item):
>                  rootmeta = root.unknowns.metadata(name)
> +
> +                if name in self._desvars:
> +                    root.unknowns.metadata(name)['is_desvar'] = True


you create a rootmeta temporary above, but then you don't use it here.

—
Reply to this email directly or view it on GitHub<https://github.com/OpenMDAO/OpenMDAO/pull/378/files#r43914562>.

---------
Herb Schilling
Scientific Applications and Visualization Team
http://ocio.grc.nasa.gov/gvis
Office of the Chief Information Officer
NASA Glenn Research Center
hschilling@nasa.gov<mailto:hschilling@nasa.gov>